ESP32LapTimer/src/ADC.cpp

```diff
@@ -70,27 +69,7 @@ void IRAM_ATTR nbADCread( void * pvParameters ) {
 
   uint32_t now = micros();
   LastADCcall = now;
-  adc1_channel_t channel = ADC1;
-  switch (current_adc) {
-    case 0:
-      channel = ADC1;
-      break;
-    case 1:
-      channel = ADC2;
-      break;
-    case 2:
-      channel = ADC3;
-      break;
-    case 3:
-      channel = ADC4;
-      break;
-    case 4:
-      channel = ADC5;
-      break;
-    case 5:
-      channel = ADC6;
-      break;
-  }
+  adc1_channel_t channel = ADC_PINS[current_adc];
```

Comment:
It should never happen, but just to be safe we should do something like this to prevent an overflow
```
adc1_channel_t channel = ADC_PINS[MIN(current_adc, MAX_NUM_RECEIVERS - 1)];
```

Reply:
Seems like a good idea to make sure that it doesn't

Reply:
Addressed in cd164a8